nkms/blockchain/eth/constants.py

```diff
@@ -1,31 +1,31 @@
 class NuCypherTokenConfig:
     __subdigits = 18
-    _M = 10 ** __subdigits
-    __premine = int(1e9) * _M
-    __saturation = int(1e10) * _M
-    _reward = __saturation - __premine
+    _M = 10 ** __subdigits                                 # Unit designation
+    __initial_supply = int(1e9) * _M                       # Initial token supply
+    __saturation = int(3.89e9) * _M                        # Token supply cap
+    _remaining_supply = __saturation - __initial_supply    # Remaining supply
 
     @property
     def saturation(self):
         return self.__saturation
 
 
 class NuCypherMinerConfig:
-    _hours_per_period = 24       # Hours
-    _min_release_periods = 30    # 720 Hours
-    __max_awarded_periods = 365
+    _hours_per_period = 24       # Hours in single period
+    _min_release_periods = 30    # 720 Hours minimum
+    __max_minting_periods = 365  # Maximum number of periods
```

Comment:
Maybe we can include a formula which explains what it is additionally?
E.g.:
```
reward_rate ~ (1 + min(lock_time, __max_minting_periods) / __max_minting_periods) / 2
```
Would it make it more understandable?